plugins/modules/ds8000_lss.py

```diff
@@ -78,24 +78,40 @@
     state: absent
 '''
 
-RETURN = r''' # '''
-# TODO this will return something
-
-LSS_TYPE = 'ckd'
-CKD_BASE_CU_TYPES = ['3990-3', '3990-tpf', '3990-6', 'bs2000']
-# The REST API returns links or even not the key. pyds8k representation returns as links or with values containing empty strings.
-REPR_KEYS_TO_DELETE = ['link', 'volumes']
+RETURN = r'''
+lss:
+    description: A list of dictionaries describing the lsses.
+    returned: I(state=present) changed
+    type: list
+    elements: dict
+    contains:
+      id:
+        description: The lss ID.
+        type: str
+        sample: "1F"
+    sample: |
```

Comment:
Interesting...didn't know you could do the sample tag on the return.

Reply:
Seems to work 
`
RETURN VALUES:
- lss
        A list of dictionaries describing the lsses.

        elements: dict
        returned: `state=present' changed
        sample: [   {     "id": "1F"   } ]
        type: list

        CONTAINS:

        - id
            The lss ID.

            sample: 1F
            type: str
`